signs-work-order-map/config/calcs.ts

```diff
@@ -0,0 +1,110 @@
+import { useEffect } from "react";
+import {
+  useCalculationType,
+  useSimpleCalculationType,
+  CalcComponentTypeDef,
+} from "@/types/calcs";
+
+export const MAXWIDTH = 12;
+export const MAXFEET = 1000;
+export const MAXTHICKNESS = 15;
+export const MAXRPMS = 100;
+
+export const defaultDivisors: { [key: string]: number } = {
```

Comment:
i am wondering if these can go into the `thermoplasticCalculations` array instead of  a separate config?